.github/workflows/ci-cd.yml

```
@@ -48,7 +94,19 @@
       run: make fmt-check
     
     - name: Run tests
-      run: go test -v -race -coverprofile=coverage.out ./...
+      run: |
+        # Start CCXT service for integration tests
+        cd ccxt-service
+        npm run dev &
+        CCXT_PID=$!
+        sleep 10
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)